app/components/org-group-form.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+
+import FormMixin from 'open-event-frontend/mixins/form';
+//OrgGroupForm
+export default Component.extend(FormMixin, {
+
+    getValidationRules() {
+        return {
+          inline : true,
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 10.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6000753)